Helper/CategorySorting.php

```diff
@@ -78,6 +78,7 @@ public function __construct(
     public static function getNostoSortingOptions()
     {
         return [
+            /** @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal */
```

Comment:
Why is this? Could you add a comment please.

Reply:
The functions `__()` expects array as a parameter in phpDoc: https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Phrase/__.php#L16

But it supports passing the string and that's how they use it throughout, so I think phpDoc could be improved from M2 side